Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -2,6 +2,13 @@ import Foundation
 
 @objcMembers
 public class SentryReplayOptions: NSObject, SentryRedactOptions {
+    
+    public enum SentryReplayQuality: Int {
+        case low
+        case medium
+        case high
```

Comment:
`m`: I think it would make sense to add some code docs to explain the difference of these as you did in the description of the PR.